src/Zadatak3/Complex.java

```diff
@@ -0,0 +1,64 @@
+package Zadatak3;
+
+public class Complex {
+
+        private int realNumber;
+        private int imaginaryNumber;
+
+        public Complex()
+        {
+            this.realNumber = 0;
+            this.imaginaryNumber = 0;
+        }
+
+        public Complex (int realNumber, int imaginaryNumber)
+        {
+                this.realNumber = realNumber;
+                this.imaginaryNumber = imaginaryNumber;
+        }
+
+        public int Re()
+        {
+                return realNumber;
+        }
+
+        public int Im()
+        {
+                return imaginaryNumber;
+        }
+
+
+        public Complex add(Complex z1)
+        {
+            int realNumber = this.realNumber + z1.realNumber;
+            int imaginaryNumber = this.imaginaryNumber + z1.imaginaryNumber;
+            return new Complex(realNumber, imaginaryNumber);
```

Comment:
no need for additional variables, the sum is straight forward enough that you can use:
`return new Complex(this.realNumber + z1.realNumber, this.imaginaryNumber + z1.imaginaryNumber);`

Also, while z1.realNumber is possible, it would be better to use z1.Re() instead:
- for one it would emphasise the difference between the parameter z1 and the current object `this`
- if z1 wasn't of type `Complex` it would have been impossible to access the attribute here, but we will learn this down the line

Also `z1` could have been called simply `z`.